Bottom: e7d4d03d082c95eafddd2b042a4fc804e0671f67
Top:    62022bd96c9c92dce6aa2ae4f4dc0f05f3e1392a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-02 18:43:08 +0000

Make --enable-dbus default


---

diff --git a/configure.ac b/configure.ac
index 0f11e4a..c1d020f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -327,7 +327,13 @@ fi
 AC_MSG_CHECKING([for whether to use DBUS])
 AC_ARG_ENABLE([dbus],
 [  --enable-dbus           Enable DBUS IPC],
-[],[enable_dbus=no])
+[],[
+	case " $with_gui " in
+		*\ gtk\ *)     enable_dbus=yes ;;
+		*\ lesstif\ *) enable_dbus=yes ;;
+		* )            enable_dbus=no ;;
+	esac
+])
 
 AC_MSG_RESULT([$enable_dbus])
 AM_CONDITIONAL(WITH_DBUS, test x$enable_dbus = xyes)
